rllib/algorithms/algorithm_config.py

```diff
@@ -500,6 +500,7 @@ def __init__(self, algo_class: Optional[type] = None):
         self.logger_config = None
         self.log_level = "WARN"
         self.log_sys_usage = True
+        self.log_gradients = True
```

Comment:
nit nit: Can you move this to the default-value block, that's headered with `reporting()`. Keeps the c'tor structured.

Reply:
Great catch! Moved it to reporting defaults.